engines/bastion_katello/app/assets/javascripts/bastion_katello/capsule-content/capsule-content.routes.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * @ngdoc object
+ * @name Bastion.capsule-content.config
+ *
+ * @requires $stateProvider
+ *
+ * @description
+ *   Used for systems level configuration such as setting up the ui state machine.
+ */
+angular.module('Bastion.capsule-content').config(['$stateProvider', function ($stateProvider) {
+
+    // Fake state to prevent redirects
```

Comment:
Sounds like this is not being handled properly in the main Bastion code, can you describe what happens? I might be able to look into it and fix it to avoid this.